README-ru.md

```diff
@@ -0,0 +1,551 @@
+- `overlayControls` — объект, который переопределяет контролы виджета в момент редактирования. Если он не передан, будут отображаться базовые контролы. При передаче `null` будет отображаться только кнопка закрытия или пользовательское меню.
+- `overlayMenuItems` — пользовательские элементы выпадающего меню.
+- `noOverlay` — если установлено значение `true`, оверлей и контролы не отображаются при редактировании.
+- `focusable` — если установлено значение `true`, элементы сетки будут доступны для фокуса.
+- `draggableHandleClassName` — имя CSS-класса элемента, который позволяет перетаскивать виджет.
+- `onDragStart` — вызывается при начале перетаскивания элемента в библиотеке `ReactGridLayout`.
+- `onDrag` — вызывается во время перетаскивания элемента в библиотеке `ReactGridLayout`.
+- `onDragStop` — вызывается по завершении перетаскивания элемента в библиотеке `ReactGridLayout`.
+- `onResizeStart` — вызывается при начале изменения размера элемента в библиотеке ReactGridLayout.
+- `onResize` — вызывается в процессе изменения размера элемента в библиотеке `ReactGridLayout`.
+- `onResizeStop` — вызывается по завершении изменения размера элемента в библиотеке `ReactGridLayout`.
+- `getPreparedCopyItemOptions` — вызывается для преобразования скопированного элемента в сериализуемый объект перед его сохранением в локальное хранилище. Используется вместо устаревшего свойства `context.getPreparedCopyItemOptions`.
+- `onCopyFulfill` — вызывается при завершении копирования элемента. При успешном копировании передается `error=null` и определенное значение `data`, при неудачном копировании — `error: Error` без `data`.
+
+## Использование
+
+### Конфигурирование DashKit
+
+Перед использованием `DashKit` в качестве React-компонента необходимо его настроить.
+
+- Установка языка:
+
+  ```js
+  import {configure, Lang} from '@gravity-ui/uikit';
+
+  configure({lang: Lang.En});
+  ```
+
+- `DashKit.setSettings`
+
+  Используется для глобальных настроек `DashKit` (отступы между виджетами, размеры виджетов по умолчанию и меню оверлея виджета).
+
+  ```js
+  import {DashKit} from '@gravity-ui/dashkit';
+
+  DashKit.setSettings({
+    gridLayout: {margin: [8, 8]},
+    isMobile: true,
+    // menu: [] as Array<MenuItem>,
+  });
+  ```
+
+- `DashKit.registerPlugins`
+
+  Регистрация и конфигурация плагинов:
+
+  ```js
+  import {DashKit} from '@gravity-ui/dashkit';
+  import {pluginTitle, pluginText} from '@gravity-ui/dashkit';
+
+  DashKit.registerPlugins(
+    pluginTitle,
+    pluginText.setSettings({
+      apiHandler({text}) {
+        return api.getMarkdown(text);
+      },
+    }),
+  );
+
+  DashKit.registerPlugins({
+    type: 'custom',
+    defaultLayout: {
+      w: 10,
+      h: 8,
+    },
+    renderer: function CustomPlugin() {
+      return <div>Custom widget with custom controls</div>;
+    },
+  });
+  ```
+
+### Конфигурация
+
+```ts
+export interface Config {
+  salt: string; // to form a unique id
```

Comment:
Looks like there should be translations for comments in code samples here and below

Reply:
We decided not to translate the comments in the code. Because, it would be to hard for cattool to parse comments from code for translation.